Apps/WebClient/src/ClientApp/src/components/navmenu/HeaderComponent.vue

```diff
@@ -82,12 +85,16 @@ export default class HeaderComponent extends Vue {
     @Ref("ratingComponent")
     readonly ratingComponent!: RatingComponent;
 
+    @Ref("appTourComponent")
+    readonly appTourComponent!: AppTourComponent;
+
     readonly sidebarId = "notification-centre-sidebar";
-    private logger!: ILogger;
```

Comment:
Why are we changing private scope on this file?

Reply:
Thanks this was cleared up by Mike there was s Volar extension issue with private values but that has since been cleared up.

I'll revert these changes as I do prefer being explicit on this. 👍